shell/platform/darwin/common/availability_version_check.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Please add a comment somewhere about why we need this and what it's doing.

Reply:
D'oh! I added the comments below locally, and then forgot to upload them...